megazords/ios-rust/Sources/MozillaRustComponentsWrapper/Nimbus/Nimbus.swift

```diff
@@ -429,81 +459,82 @@ extension Nimbus: NimbusMessagingProtocol {
 public class NimbusDisabled: NimbusApi {
     public static let shared = NimbusDisabled()
 
-    public var globalUserParticipation: Bool = false
+    public var experimentsUserParticipation: Bool = true
+    public var rolloutsUserParticipation: Bool = true
 }
 
-public extension NimbusDisabled {
-    func getActiveExperiments() -> [EnrolledExperiment] {
+extension NimbusDisabled {
+    public func getActiveExperiments() -> [EnrolledExperiment] {
         return []
     }
 
-    func getAvailableExperiments() -> [AvailableExperiment] {
+    public func getAvailableExperiments() -> [AvailableExperiment] {
         return []
     }
 
-    func getExperimentBranch(experimentId _: String) -> String? {
+    public func getExperimentBranch(experimentId _: String) -> String? {
         return nil
     }
 
-    func getVariables(featureId _: String, sendExposureEvent _: Bool) -> Variables {
+    public func getVariables(featureId _: String, sendExposureEvent _: Bool) -> Variables {
         return NilVariables.instance
     }
 
-    func initialize() {}
+    public func initialize() {}
 
-    func fetchExperiments() {}
+    public func fetchExperiments() {}
 
-    func setFetchEnabled(_: Bool) {}
+    public func setFetchEnabled(_: Bool) {}
 
-    func isFetchEnabled() -> Bool {
+    public func isFetchEnabled() -> Bool {
         false
     }
 
-    func applyPendingExperiments() -> Operation {
+    public func applyPendingExperiments() -> Operation {
```

Comment:
Not sure what happened here. Might be the Swift PlugIn in my VScode did some shenanigans?